### PR TITLE
feat(Table): get rid of gravity usage in the base table

### PR DESCRIPTION
**Pull request by beliarh**

### CODE REVIEW

File: src/components/Table/Table.scss [hunk 1]
```diff
@@ -48,39 +36,37 @@ $block: '.#{variables.$ns}table';
...
+            transform: rotate(0);
+        }
     }
 
     &__group-expand-button {
         appearance: none;
 
         display: flex;
-        gap: var(--g-spacing-2);
+        gap: 8px;
```
Comment (chelentos) [inline src/components/Table/Table.scss:69]: Maybe we should create some css variables for spacing?

File: src/components/__stories__/ColumnPinningDemo.classname.ts [hunk 2]
```diff
@@ -1,3 +1,3 @@
-import {block} from '../../utils';
+import {cn} from '../../utils';
 
-export const cnColumnPinningDemo = block('column-pinning-demo');
+export const cnColumnPinningDemo = cn('column-pinning-demo');
```
Comment (chelentos) [inline src/components/__stories__/ColumnPinningDemo.classname.ts:3]: Why was this change necessary?

File: src/components/Table/Table.scss (hunk 1 above)
Comment (beliarh) [reply src/components/Table/Table.scss:69]: There is no need. This component doesn't have any specific design. New styled table will use gravity/uikit styles and variables.

File: src/components/__stories__/ColumnPinningDemo.classname.ts (hunk 2 above)
Comment (beliarh) [reply src/components/__stories__/ColumnPinningDemo.classname.ts:3]: There are no reasons to add `gt-` prefix for demo components.
That's why we use `cn` in stories.

**Review by kseniya57 — APPROVED** [review]
